plugins/formal-verification/formver.common/src/org/jetbrains/kotlin/formver/ErrorManager.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver
+
+class ErrorManager {
```

Comment:
I would call this class `ErrorCollector` compared to manager. So the meaning of the class can be understood easily while reading it in the code.